src/App.js

```diff
@@ -80,23 +93,29 @@ export default function App(props) {
   }
 
   return (
-    <div className="root">
+    <div className="root" style={{ overflow: 'hidden', height: '100vh' }}>
       <TopBar />
       <SUSHIControls onClick={handleSUSHIControls} text={inputText} resetLogMessages={resetLogMessages} />
       <Grid className={classes.container} container>
-        <Grid className={classes.itemTop} item xs={6}>
+        <Grid style={{ height: expandConsole ? '55vh' : '85.3vh' }} item xs={6}>
```

Comment:
Is there a reason to move these from the classes to a `style` tag? I think generally we want to keep styles in the classes if we can.

Similarly for the `Grid` below this one.

Reply:
I was using the `style` tag because I wanted to make the height of many of the components dependent on the `expandConsole` property that's created in the main app, which I didn't think I could do in the classes since that variable arrives as part of the `props` arguments in the ConsoleComponent. I think I could just move the styles object down a level into the main `App` function and that should be a workaround.

Evidently that may not be what we want to do anyway, I'm beginning to think that these ternary statements aren't the best way to approach resizing the components on the screen 😕.